2006Redone Client/src/ClientSettings.java

```diff
@@ -18,8 +18,8 @@
 	/**
 	 * The Servers Ip
 	 */
-	public static String SERVER_IP = "35.226.247.68";
-	//public final static String SERVER_IP = "127.0.0.1";
+	public static String SERVER_IP = "127.0.0.1";

```

Comment:
Undo this change - if you run from Client.java it automatically connects locally, or if you run with two random arguments from Main.java it will autoconnect local. I'll make this more clear (or you can add this as a comment on this line and that's :+1:  too)